hub/src/server/revocations.js

```diff
@@ -0,0 +1,98 @@
+/* @flow */
+
+import LRUCache from 'lru-cache'
+import type { DriverModel } from './driverModel'
+import fetch from 'node-fetch'
+import { Readable } from 'stream'
+
+const MAX_AUTH_FILE_BYTES = 1024
+const AUTH_TIMESTAMP_FILE_NAME = 'authTimestamp'
+
+export class AuthTimestampCache {
+
+  cache: LRUCache;
+  driver: DriverModel;
+
+  constructor(driver: DriverModel) {
+    // TODO: configure cache settings..
+    this.cache = new LRUCache()
```

Comment:
How big can this cache get?  Can it exceed the size of RAM?

Reply:
Also, just want to confirm -- the `LRUCache` is _only_ in-RAM, right?

Reply:
Reasonable cache max default has been implemented, with eviction warning logging, and tests to ensure cache eviction is working as expected.